msbuild/Xamarin.iOS.Tasks.Windows/Properties/Resources.resx

```diff
@@ -19,4 +19,7 @@
 		<value>Info.plist file does not exist under {0}, please rebuild your project</value>
 		<comment>0: path</comment>
     </data>
+	<data name="HotRestartStaticLibraryNotSupported" xml:space="preserve">
```

Comment:
Is the values inside this resx file something that we'd want localized?

Reply:
Yes, probably